packages/babel-plugin-transform-es2015-classes/src/loose.js

```diff
@@ -6,14 +6,33 @@ export default class LooseClassTransformer extends VanillaTransformer {
   constructor() {
     super(...arguments);
     this.isLoose = true;
+    this.aliasInserted = false;
+    this.methodAlias = "_proto";
+  }
+
+  _insertProtoAliasOnce() {
+    if (!this.aliasInserted) {
+      const classProto = t.memberExpression(this.classRef, t.identifier("prototype"));
+      const protoDeclaration = t.variableDeclaration("var", [
+        t.variableDeclarator(t.identifier(this.methodAlias), classProto),
```

Comment:
Thanks, that makes sense.

would replacing line 10: 
```Javascript
this.methodAlias = "_proto";
``` 
with 
```Javascript
this.methodAlias = this.path.scope.generateUidIdentifier("proto").name;
```

be a good way of achieving this?